.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+This template is for bug reports. If you are here for another reason, please see below:
+
+1. To propose a new rule, see http://eslint.org/docs/developer-guide/contributing/new-rules
+2. To request a change, see http://eslint.org/docs/developer-guide/contributing/changes
```

Comment:
You don't think they'll narrow in on the heading which applies, and only read / fill out the fields under that heading?